app/src/main/java/io/github/dotslash21/faclient/AuthActivity.java

```diff
@@ -42,6 +41,8 @@
 import io.github.dotslash21.faclient.utils.GraphicOverlay;
 import io.github.dotslash21.faclient.utils.FaceIdentificationProcessor;
 
+import static android.hardware.Camera.*;
```

Comment:
What is the need for this?? `import android.hardware.Camera` worked fine..

Reply:
resolved